packages/react/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "gcds-react-components",
```

Comment:
How do you feel about renaming the package to `gcds-components-react` instead? This way if a user searches for `gcds-components` all packages will show up.